### ISSUE TITLE
Nvda is not shown in the kodi screen reader settings after installing 17.6.  This occurs even after installing the screenreader again and running the nvda controller client script

### ISSUE THREAD

**Issue by indabs** [ruuk/service.xbmc.tts#18]
It works fine with 16.1 but once kodi is updated to 17.6, nvda is not a choice in the engine settings.

**Comment by indabs** [comment]
This continues through version 19.   Nvda doesn't work even after installng the client.  The kodi screen reader works but not nvda.  Nvda sows as enabled but as an orphan under dependantcies.
